src/worker.py

```
@@ -6646,6 +6670,436 @@ def _clamp_01(value):
+                dec_payload = await decrypt_aes(row.payload or "", enc)
+                msg_obj = json.loads(dec_payload) if dec_payload else {}
+                fields = msg_obj.get("fields") or {}
+                m_thread_id = str(fields.get("thread_id") or "")
+                m_sender = str(fields.get("sender") or fields.get("from_user_id") or fields.get("author") or "")
+                m_receiver = str(fields.get("receiver") or fields.get("to_user_id") or "")
+
+                is_match = False
+                if m_thread_id and m_thread_id == str(thread_id):
+                    is_match = True
+                elif (m_sender == user["id"] and m_receiver == other_id) or (m_sender == other_id and m_receiver == user["id"]):
+                    is_match = True
```

Ruff (0.15.21)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)